src/index.js

```diff
@@ -136,7 +136,9 @@ class FsDatastore {
       throw new Error(`Invalid extension: ${path.extname(file)}`)
     }
 
-    return new Key(file.slice(this.path.length, -ext.length))
+    let keyname = file.slice(this.path.length, -ext.length)
```

Comment:
`s/let/const`

Reply:
Its an age thing.  I come from AI/functional programming.  I only use const for scalars.  Consider my hand slapped.

Reply:
I'm a newbie to organisational github.  How to I change a PR that you submitted?  I'm use to being on the other side.

Reply:
The simpler way is to clone again.

```
git clone git@github.com:ipfs/js-datastore-fs.git
cd js-datastore-fs
git checkout windows-interop
# do your changes
git add <files changed>
git commit -m "message"
git push origin windows-interop
```